core/store/orm/orm.go

```diff
@@ -1173,6 +1175,44 @@ func (orm *ORM) FindLogCursor(name string) (models.LogCursor, error) {
 	return lc, err
 }
 
+// HasConsumedLog ...
```

Comment:
Docs would be great, maybe even just mark with TODO. We can do theses in a subsequent PR if it'd help.